sys/net/gnrc/network_layer/ipv6/nib/_nib-router.c

```diff
@@ -215,6 +215,52 @@ static gnrc_pktsnip_t *_build_ext_opts(gnrc_netif_t *netif,
     return ext_opts;
 }
 
+/* Sending a RA with ltime = 0 causes the router to be removed from the
+ * default router list, but options are still parsed.
+ * This allows us to add downstream subnets that should be routed through
+ * this router, but the router is not an upstream / default router for
+ * this link. */
+static gnrc_pktsnip_t *_build_final_ext_opts(gnrc_netif_t *netif)
+{
+    gnrc_pktsnip_t *ext_opts = NULL;
+    _nib_offl_entry_t *entry = NULL;
+
+    if (!IS_USED(MODULE_GNRC_IPV6_NIB_RIO) ||
+        !IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_RIO_IN_LAST_RA)) {
+        return NULL;
```

Comment:
So the final RA will now have no options at all when `!IS_ACTIVE(CONFIG_GNRC_IPV6_NIB_RIO_IN_LAST_RA)`?

Reply:
Yes this is the current behavior in master.
No options are added in the `final` case.